chainer/functions/array/reshape.py

```diff
@@ -88,7 +88,7 @@ def reshape(x, shape):
         ...
         chainer.utils.type_check.InvalidType:
         Invalid operation is performed in: Reshape (Forward)
-
+        <BLANKLINE>
```

Comment:
Is it really required? In my environment this test can be passed.

Reply:
I read this [document](https://docs.python.org/2/library/doctest.html).
> Expected output cannot contain an all-whitespace line, since such a line is taken to signal the end of expected output. If expected output does contain a blank line, put <BLANKLINE> in your doctest example each place a blank line is expected.
> New in version 2.4: <BLANKLINE> was added; there was no way to use expected output containing empty lines in previous versions.

Reply:
The test can pass even without this fix. Is this code really tested?

Reply:
I tested some variations and found that `...` matches all and it ignores logs after `...`. So this fix is also ignored. But ok i'll merge it.